e2e/README.md

```diff
@@ -0,0 +1,39 @@
+# Guide to Running Tests
+
+It is important to note that one must ensure that all dfusion related builds are recent (i.e. driver, graph-listener and their dependents). 
```

Comment:
What do you mean "related builds are recent", are these the docker images? Maybe include instructions on how to do that?

Reply:
+1. I am unable to run the tests successfully, probably doing something wrong related to this.

Reply:
Right yes, this would require following the main project readme guide from cloning to building. I can include a reference to this, but not sure it makes sense to include word for word twice. 

@e00E - Your issue is probably something to do with the `dex-contracts` submodule setup IN particular, you might require a couple of the following commands from the main readme. 

```
git submodule update --init
cd dex-contracts && yarn && yarn prepack && cd ..
```

You will notice that I have not include any migrations, or manually running `ganache` because these are included in the truffle container. From the test plan.